app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/theme/Color.kt

```diff
@@ -0,0 +1,17 @@
+package hr.dice.filipbionda.tmdbpractice.ui.theme
+
+import androidx.compose.ui.graphics.Color
+
+val Purple80 = Color(0xFFD0BCFF)
+val PurpleGrey80 = Color(0xFFCCC2DC)
+val Pink80 = Color(0xFFEFB8C8)
+
+val Purple40 = Color(0xFF6650a4)
+val PurpleGrey40 = Color(0xFF625b71)
+val Pink40 = Color(0xFF7D5260)
+
+val welcomeScreenPrimaryTextColor = Color.White
```

Comment:
Call the colors as they are. For example white, transparent, black_70 (70 is a ramp value and represents 70% alpha), etc.

Reply:
done